tests/integration/it_utils.py

```diff
@@ -302,3 +308,143 @@ def _get_non_decorator_code(function):
         if "def " in code_lines[line]:
             return code_lines[line:]
         line += 1
+
+
+def setup_amm_pool(
```

Comment:
This shouldn't be in `it_utils` if it's only used in `reusable_values` - it should just be there (and you only need one version of this method that way).

Reply:
`it_utils` is only needed for methods that are being used in the actual tests.